test_unstructured/metrics/test_text_extraction.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from unstructured.metrics.text_extraction import calculate_edit_distance
+from unstructured.partition.auto import partition
+
+
+@pytest.mark.parametrize(
+    ("filename", "expected_score", "expected_distance"),
+    [
+        ("fake-text.txt", 0.82, 30),
+        ("eml/fake-email.eml", 0.03, 164),
+        ("README.md", 0.0, 719),
+    ],
+)
+def test_calculate_edit_distance(filename, expected_score, expected_distance):
+    with open("example-docs/fake-text.txt") as f:
+        source_cct = f.read()
+
+    elements = partition(filename=f"example-docs/{filename}")
+    output_cct = "\n\n".join([str(el) for el in elements])
```

Comment:
let's join with a single "\n" instead of two

Reply:
Do we want to follow the demo on the unstructured's main page? The demo uses `\n\n` so I followed that. Also, I believe this function should be edited once the CCT function is ready. So once this is done, we'll want to create a ticket to edit this test case :)

Reply:
The demo is just to print reasonably, but for this string comparison purpose a single newline or even single space will suffice. 
Yes, I agree we can edit this once we have a CCT function !